### DISCUSSION TITLE
switch from json to ordered_json

### DISCUSSION THREAD

**Discussion by gkriger** [nlohmann/json#3342] [Q&A] [answered]
From the perspective of casual user the most natural way of switching the code base to ordered_json seems to be replacing
```cpp
using json = nlohmann::json;
```
with
```cpp
using json = nlohmann::ordered_json; 
```
And the things mostly work. However, this:
```cpp
json::object_t jobj({ { "product", "one" } });
```
does not compile however it compiles just fine with using json = nlohmann::json;
Compiler: Microsoft Visual C++ Compiler 16.4.29609.76 (x86_amd64) with c++11 flag.
lib release version 3.10.5

**Comment by nlohmann** [comment, marked as answer]
See #3343 - fixed with #3370.
